src/sky.cpp

```diff
@@ -537,6 +581,44 @@ void Sky::render()
 			}
 			driver->drawIndexedTriangleFan(&vertices[0], 4, indices, 2);
 		}
+
+		// Draw upper blend fog band
+		for (u32 j = 0; j < 2; j++) {
+			video::SColor c = m_bgcolor_alpha;
+			video::SColor c_blend = m_skycolor_alpha; //
+				vertices[0] = video::S3DVertex(-1, -0.1, -1, 0, 0, 1, c, t, t);
+				vertices[1] = video::S3DVertex( 1, -0.1, -1, 0, 0, 1, c, o, t);
+				vertices[2] = video::S3DVertex( 1, 0.3, -1, 0, 0, 1, c_blend, o, o);
+				vertices[3] = video::S3DVertex(-1, 0.3, -1, 0, 0, 1, c_blend, t, o);
+			for (video::S3DVertex &vertex : vertices) {
+				if (j == 0)
+					// Switch from -Z (south) to +X (east)
+					vertex.Pos.rotateXZBy(90);
+				else if (j == 1)
+					// Switch from -Z (south) to -X (west)
+					vertex.Pos.rotateXZBy(-90);
+			}
+			driver->drawIndexedTriangleFan(&vertices[0], 4, indices, 2);
+		}
+
+		// Draw bottom sides of the sky, to hide the sunrise texture
+		for (u32 j = 0; j < 2; j++) {
+			video::SColor c = m_bgcolor;
```

Comment:
Please use `cloudyfogcolor` as before for clarity.

Reply:
Renaming `m_bgcolor` and `m_skycolor` to `m_sky_color_bottom` and `m_sky_color_top` would be better since `cloudyfogcolor` was used once to calculate the difference between the top half and the bottom half the sky.

Reply:
Yes using new and more suitable names might be a good idea.